homeassistant/components/binary_sensor/translations/en.json

```diff
@@ -195,8 +193,8 @@
             "on": "Plugged in"
         },
         "presence": {
-            "off": "Away",
-            "on": "Home"
+            "off": "[%key:common::state::not_home%]",
+            "on": "[%key:common::state::home%]"
```

Comment:
Why does this change after running:
`python3 -m script.translations develop --integration binary_sensor`

Happy to revert manually, but would be good to understand if I need to run another command.

Reply:
Bug in our resolving script unable to handle double references. Pushed a fix.